miden-tx/src/host/mod.rs

```diff
@@ -11,14 +11,14 @@ use miden_objects::{
     Digest, Hasher,
 };
 use vm_processor::{
-    crypto::NodeIndex, AdviceExtractor, AdviceInjector, AdviceProvider, AdviceSource, ContextId,
-    ExecutionError, Felt, Host, HostResponse, ProcessState,
+    AdviceExtractor, AdviceInjector, AdviceProvider, AdviceSource, ContextId, ExecutionError, Felt,
+    Host, HostResponse, ProcessState,
 };
 
 mod account_delta_tracker;
 use account_delta_tracker::AccountDeltaTracker;
 
-mod account_procs;
+pub mod account_procs;
 use account_procs::AccountProcedureIndexMap;
```

Comment:
nit: rather than exporting the module, we could probably export just the struct. That is, this could be:
```rust
mod account_procs;
pub use account_procs::AccountProcedureIndexMap;
```

Reply:
Changed.